services/request/index.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+const got = require('got');
+const merge = require('lodash.merge');
+
+function createRequestService() {
+    function post(options) {
+        let opts = {
+            method: 'POST',
+            headers: {
+                accept: 'application/vnd.api+json',
+                'Content-Type': 'application/vnd.api+json'
+            },
+            json: true,
+            body: options.body
+        };
+        opts = merge(opts, options);
+        return got(options.url, opts);
+    }
+
+    function get(options) {
+        let opts = {
+            method: 'GET',
+            headers: {
+                accept: 'application/json',
+                'Content-Type': 'application/json'
```

Comment:
Should this be just `application/json` and not `application/vnd.api+json`

Reply:
well caught! I'll add it in

Reply:
This is being handled in the `message-bus` service file!